pkgs/jnigen/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 0.8.0-wip
 
+- **Breaking Change** ([#981](https://github.com/dart-lang/native/issues/981)):
+  - `fromRef` is now `fromReference`, and it gets a `JReference` instead of a
+    `Pointer<Void>`.
+  - Check out the internal changes to `JObject` in
+    [`package:jni`'s changelog](https://github.com/dart-lang/native/blob/main/pkgs/jni/CHANGELOG.md#080-wip).
```

Comment:
This link will be out of date after we publish 0.8.0 non WIP.

Consider either leaving no anchor, or make a note somewhere to update it.

Reply:
Local note should be fine.